packages/backend-plugin-api/src/services/definitions/coreServices.ts

```diff
@@ -226,6 +239,19 @@ export namespace coreServices {
     import('./RootLoggerService').RootLoggerService
   >({ id: 'core.rootLogger', scope: 'root' });
 
+  /**
+   * Root-level event auditing.
+   *
+   * See {@link RootEventAuditorService}
+   * and {@link https://backstage.io/docs/backend-system/core-services/root-event-auditor | the service docs}
+   * for more information.
+   *
+   * @public
+   */
+  export const rootEventAuditor = createServiceRef<
+    import('./RootEventAuditorService').RootEventAuditorService
+  >({ id: 'core.rootEventAuditor' }); // Do we need `scope: root` here?
```

Comment:
Yes, this needs root scoping

Reply:
This is currently not feasible because `auth` and `httpAuth` are designated as dependencies, meaning `rootAuditorService` cannot be changed to `scope: root`. How should I proceed?

Reply:
Alright. We've been discussing whether a root token management service or so could make sense to add to the toolbox, but no immediate plans.

I'd say, for now, I'd vote to skip the root version of the auditor. It can always be added later on down the line. @Rugvip?

Reply:
> I'd say, for now, I'd vote to skip the root version of the auditor. It can always be added later on down the line. @Rugvip?

Skipping the root implementation might not be the best approach, as we only want a single logging instance for the auditor service, similar to how the logging service operates with just one instance.

Reply:
You can do that for plugin scoped things. See what [the cache service does](https://github.com/backstage/backstage/blob/e7ce581513864aa32443e24fbeb84c11b34d704c/packages/backend-defaults/src/entrypoints/cache/cacheServiceFactory.ts#L39) for example. `createRootContext` is guaranteed to be called only once. This structure isn't exposed to callers, and can be refactored into two tiers of service later down the line without affecting them.